hub/subscribe.go

```diff
@@ -202,3 +233,53 @@ func (h *Hub) cleanup(s *Subscriber) {
 	}
 	h.uriTemplates.Unlock()
 }
+
+func (h *Hub) dispatchSubscriptionUpdate(topics, encodedTopics []string, connectionID string, claims *claims, active bool, address string) {
+	if !h.config.GetBool("dispatch_subscriptions") {
+		return
+	}
+
+	for k, topic := range topics {
+		connection := &subscription{
+			ID:      "https://mercure.rocks/subscriptions/" + encodedTopics[k] + "/" + connectionID,
```

Comment:
Put `https://mercure.rocks/subscriptions` or at least `https://mercure.rocks` in constant or in env var setup

Reply:
https://github.com/dunglas/mercure/pull/238#discussion_r380677961

It's a fixed value, on purpose so an env var isn't applicable. IMHO using a constant isn't worth it because every value is used only 1 time.